app/src/components/TextField.tsx

```diff
@@ -0,0 +1,56 @@
+import { Trans } from 'next-i18next'
+import { ChangeEvent, ReactElement } from 'react'
+
+import Required from '@components/Required'
+
+import { i18nKey } from '@src/types'
+
+export type TextFieldProps = {
+  handleChange: (
+    e: ChangeEvent<HTMLInputElement> | ChangeEvent<HTMLTextAreaElement>
```

Comment:
Notes for reviewers: This is a new component refactored out of TextArea and TextInput. I originally created those two components because I was having trouble with typescript typing for the `handleChange` prop. I was able to smoosh them back into one component because I figured out the correct way to type the prop.